generators/rest-api/templates/src/main/java/package/web/rest/ClientResource.java.ejs

```diff
@@ -22,7 +22,7 @@ public class ClientResource<%= capitalizeServerName %> {
 
     @Autowired
     private WebClientConfig webClient;
-
+z
```

Comment:
remove this alphabet "z" from the template file!
application wont be able to build with such characters in between.

Reply:
Sorry my mistake I'll remove it 😅